include/ccf/http_configuration.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/ds/json.h"
+#include "ccf/ds/unit_strings.h"
+
+#include <optional>
+
+namespace http
+{
+  struct ParserConfiguration
+  {
+    ds::SizeString max_body_size = {"1MB"};
+    ds::SizeString max_header_size = {"16KB"};
+    size_t max_headers_count = 256;
```

Comment:
The default values for these entries are:
- `max_body_size`: `1MB` is what we've tested in the CI for a very long time now.
- `max_header_size`: NodeJS default value.
- `max_headers_count`: roughly guided by the two other values, and would leave 256 headers with a size of 4KB each (2KB for each key and value) to end up with a total headers size the size of the total body size.

Reply:
These values and the fact they can be set should be touched upon in the operator doc.

Reply:
Documentation for these is automatically generated [here](https://microsoft.github.io/CCF/main/operations/configuration.html). Do you mean something more specific?